### ISSUE TITLE
Feature: Edit data in content view.

### ISSUE THREAD

**Issue by markmorris** [Alecaddd/sequeler#164]
Be able to edit the data when viewing the content of a table. Once that is done, i can completely replace MySQL Workbench and would be willing to pay twice for this software.

Keep up the good work!

**Comment by Alecaddd** [comment]
Yup, this feature is scheduled for the next release.
Thank you so much for using Sequeler, I built this app exactly for the same reason as I use it every day 😄 

I'm closing this as it's a duplicate of #67
